.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright © Idiap Research Institute <contact@idiap.ch>
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+# dependabot.yaml reference: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  # Maintain dependencies in our GitHub Workflows
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: monthly
+      time: "05:00"
+      timezone: CET
```

Comment:
According to the [github docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups), by default, dependabot will make one pull request for **each** dependency that needs an update at the current run. The `groups` config allows the merge of multiple dependencies needing update into fewer pull requests, following a few grouping rules.

It only affects Dependabot's behavior when multiple actions have an update during the same run.
- The advantage of grouping is that fewer pull requests are made; and if two dependencies update at the same time and need each other to be updated, then manual intervention is not necessary;
- The disadvantage is that it could [make a large pull request](https://github.blog/changelog/2023-06-30-grouped-version-updates-for-dependabot-public-beta/).

That particular configuration from gridtk groups everything together. So all GitHub Actions dependencies we have that require an update will be put together in one pull request. I don't think it would be too bad as we don't have that many.

I will add it here too.